### PR TITLE
Issue/414

### PR DESCRIPTION
**Pull request by alexsasharegan**
This should provide support for the basic `navbar-toggleable` class.

From issue #414

### CODE REVIEW

**Comment by alexsasharegan** [comment]
Actually, I need to change this. Null will still trigger the join with a hanging hyphen.

**Comment by alexsasharegan** [comment]
We'll call this WIP.

File: lib/components/navbar.vue [hunk 1]
```diff
@@ -33,7 +36,7 @@
             },
             toggleBreakpoint: {
                 type: String,
-                default: 'sm'
```
Comment (pi0) [inline lib/components/navbar.vue]: What if `toggleable` is `true` and user not providing `toggleBreakpoint` which is null by default ? :))

File: lib/components/navbar.vue [hunk 2]
```diff
@@ -15,8 +15,11 @@
                     this.variant ? `bg-${this.variant}` : null,
                     this.fixed ? `fixed-${this.fixed}` : null,
                     this.sticky ? 'sticky-top' : null,
-                    this.toggleable ? `navbar-toggleable-${this.toggleBreakpoint}` : null
+                    this.toggleable ? this.toggleClass : null
                 ];
+            },
+            toggleClass() {
+                return ['navbar-toggleable' + this.toggleBreakpoint].join('-');
```
Comment (pi0) [inline lib/components/navbar.vue]: if `toggleBreakpoint` is null it will return `nav-toggleable-` which is wrong

Comment (alexsasharegan) [inline lib/components/navbar.vue]: Yeah, I mentioned that above. I put this together a little too hastily between today's hectic schedule.

File: lib/components/navbar.vue (hunk 1 above)
Comment (alexsasharegan) [inline lib/components/navbar.vue]: I believe that's the direction I ended up taking this. Let me know!

**Comment by pi0** [comment]
@alexsasharegan Is your work done?

**Comment by alexsasharegan** [comment]
Yes. Tested all the scenarios I can think of, and it looks good!

**Pull request merged**